README.md

```diff
@@ -72,13 +71,14 @@ from iopipe.iopipe import IOpipe
 def handler(event, context):
   iopipe = IOpipe(CLIENT_ID)
   timestamp = time.time()
+  iopipe.start_report(timestamp, context)
```

Comment:
Yeah I could see that ... and make it so the parent module is _just_ responsible for config?